### DISCUSSION TITLE
Why app is rendered twice on first request?

### DISCUSSION THREAD

**Discussion by 07akioni** [nuxt/nuxt#17438] [Questions]
I note that `pages/index.vue`'s `setup` is called twice on first request (after dev server is started).

After that each refresh only cause 1 `setup` invoked.

This is not relevant with favicon. I print `nuxtApp.ssrContext.url` in a plugin. First 2 requests' url are both `/`.

<img width="356" alt="image" src="https://user-images.githubusercontent.com/18677354/168649803-c1000f7d-e6ef-4fb3-9796-59d741250c6e.png">

**Comment by byf77** [comment]
Do you have a favicon?
because if not, it will give error 404 and nuxt start another instance for render the page error

**Reply by 07akioni** [reply]
I'm sure favicon is not relevant.

**Reply by 07akioni** [reply]
url doesn't point to `/favicon`

**Comment by Flanker72** [comment]
I have such an issue a couple of times with opened Chrome DevTools after some hours of working with HMR. After closing/reopening the problem went away.

This may also be related to some tabs opened in the background. Check that you haven't any tab in the background with localhost:3001 opened.
